joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50 

front_right_controller:
  type: effort_controllers/JointVelocityController
  joint: front_right_wheel_joint  
  pid: {p: 10.0, i: 0.01, d: 0.0}

front_left_controller:
  type: effort_controllers/JointVelocityController
  joint: front_left_wheel_joint  
  pid: {p: 10.0, i: 0.01, d: 0.0}

rear_right_controller:
  type: effort_controllers/JointVelocityController
  joint: rear_right_wheel_joint  
  pid: {p: 10.0, i: 0.01, d: 0.0}

rear_left_controller:
  type: effort_controllers/JointVelocityController
  joint: rear_left_wheel_joint  
  pid: {p: 10.0, i: 0.01, d: 0.0}
